backend/server.ts

```diff
@@ -30,4 +30,20 @@ app.get("/gardens", async (req:Request , res:Response) => {
   res.send(gardens)
 })
 
+//updates a garden
+app.put("/gardens/:id", async (req:Request, res: Response) => {
+  const GardenName = req.params.recipeName
+  const description = req.body.description
```

Comment:
for this endpoint, we're assuming everything attribute in Garden is getting updated, so update all the attributes from #1

Reply:
Hi I made the changes requested and re committed the updated files!